### ISSUE TITLE
Motif not being assigned to a community

### ISSUE THREAD

**Issue by luiztauffer** [EthoML/VAME#87]
When grouping motifs into communities (`vame.community()`), depending on the value of `cut_tree`, some motifs are left out and not grouped in any of the created groups. Changing `cut_tree` to lower values seems to resolve this. See examples below:

For `cut_tree=3`:  (motifs 0, 1, 4, 10 are left out)
```
Communities bag:
Community 0: [8, 12, 7]
Community 1: [14, 13]
Community 2: [9, 11, 5]
Community 3: [3, 6, 2]
```

For `cut_tree=2`: (all motifs are included)
```
Communities bag:
Community 0: [1, 8, 12, 7]
Community 1: [4, 10]
Community 2: [14, 13, 9, 11, 5]
Community 3: [0, 3, 6, 2]
```

In the interactive mode, vame asks the user how they want to incorporate each of the motifs left outside into the community bags, one by one. This of course can’t be used in a fully automatized pipeline. 

If any motifs are left out of the community_bag, they will be automatically considered as being part of the community 0, by the current implementation of [get_cohort_community_labels](https://github.com/EthoML/VAME/blob/0dc5f138dccec393d5b420fa0c44b0f16bab9bee/src/vame/analysis/community_analysis.py#L385-L415). I believe this is wrong and should be fixed.

So my question is: given a value for `cut_tree`, how the algorithm should allocate the motifs that are left out from the tree cut?

Tree for the examples above:
![image](https://github.com/user-attachments/assets/50e3f270-74d2-4198-9300-feb25cdba99b)

I imagine two possible solutions to automatize this, if any motifs are left out of the community bags:

1. the `community()` function re-runs with a lower value for `cut_tree`.

2. each one of the left-out motifs are allocated to a new community, where they are the only motif

The number 2 sounds more reasonable to me, but I need your decision on this.

**Comment by DrSRMiller** [comment]
@luiztauffer No, this is not expected. All motifs should be assigned to a node in the graph and present in community bag. Do you know why it may be being dropped?

**Comment by luiztauffer** [comment]
@DrSRMiller I re-wrote the issue to be more explicit about what is happening. I imagine two possible solutions to automatize this, if any motifs are left out of the community bags:

1. the `community()` function re-runs with a lower value for `cut_tree`.

2. each one of the left-out motifs are allocated to a new community, where they are the only motif

The number 2 sounds more reasonable to me, but I need your decision on this

**Comment by katiekly** [comment]
@luiztauffer I agree with option 2, but in this example I would have motif 4 and 10 be in one community vs each in a separate community. @DrSRMiller would you agree?

**Comment by luiztauffer** [comment]
@katiekly but joining them would be the same as the approach number 1, right? Which criteria should we use to decide on that?

**Comment by DrSRMiller** [comment]
@luiztauffer @katiekly I've just discussed this with Pranav and I think that strategy #2 matches how we have approached this issue in the past. For example in [Figure 7A in the Cell Reports pape](https://www.cell.com/cell-reports/fulltext/S2211-1247(24)01221-X#fig7), Motifs 17 and 22 belong to their own communities. Community A was conjoined with B1 only after manual review of fine kinematics, motif and community videos, and disease effects.
